scripts/common/docker.sh

```diff
@@ -35,8 +35,9 @@ EOF
 function install_docker() {
     if [ "$SKIP_DOCKER_INSTALL" != "1" ]; then
         if [ -z "$DOCKER_VERSION" ]; then
-            printf "${RED}The installer did not specify a version of Docker to include, but is required by all kURL installation scripts currently. The latest supported version of Docker will be installed.${NC}\n"
-            DOCKER_VERSION="19.03.4"
+            printf "${YELLOW}The installer did not specify a version of Docker or Containerd to include, but having one is required by all kURL installation scripts. The latest supported version of Docker will be installed.${NC}\n"
+            logStep "Installing docker version 20.10.17"
```

Comment:
Do we need this logstep?

Reply:
isn't it redundant?

Reply:
I think would be nice we know the version that will be installed. 
So that, we might get that we are not installing the latest for example or that some issue might be related to within.

Reply:
If you have a better wording suggesting and etc please feel free to raise the idea.

Reply:
@laverya I tried to address this one, let me know wdyt?